src/utils/rewriteImports.js

```diff
@@ -17,7 +17,8 @@ export default (error, file, contents, moduleContext, callback) => {
   const rewritten = contents.replace(importRegexp, (entire, single, double, unquoted) => {
     const oldImportPath = single || double || unquoted;
     const absoluteImportPath = path.join(path.dirname(file), oldImportPath);
-    const newImportPath = path.relative(moduleContext, absoluteImportPath);
+    const relImportPath = path.relative(moduleContext, absoluteImportPath);
+    const newImportPath = relImportPath.split(path.sep).join('/');
```

Comment:
@justin808 I don't think a direct call to a file system is suitable here. Using `@import`s from within SCSS files slows down builds, a direct call to fs will make builds even slower.

@bjnsn @justin808 All in all, this PR looks good to me. So 👍  from me.

Also, see https://github.com/shakacode/sass-resources-loader/issues/46